app/assets/main/react_components/translations/react.sv.yml

```diff
@@ -41,7 +41,7 @@ sv:
         icon: eye
       projects:
         title: Projekt som gör skillnad
-        text: Med hjälp av vår kompetens handplockar vi de projekt som ger mest klimatnytta
+        text: Vi väljer våra projekt noggrant utifrån mest klimatnytta för pengarna
```

Comment:
Jag vi borde se över copy här för båda språken
Engelska: "by cost" skaver, kanske "benefit given climate impact and cost" eller nått sånt
Svenska: "utifrån", jag gillar förra bättre, men ska inte försöka komma på något nu, kl 17